android/quest/src/main/java/org/smartregister/fhircore/quest/data/QuestXFhirQueryResolver.kt

```diff
@@ -16,16 +16,80 @@
 
 package org.smartregister.fhircore.quest.data
 
+import android.util.Log
 import com.google.android.fhir.FhirEngine
 import com.google.android.fhir.datacapture.XFhirQueryResolver
+import com.google.android.fhir.search.Search
 import com.google.android.fhir.search.search
+import org.hl7.fhir.r4.model.QuestionnaireResponse
 import javax.inject.Inject
 import javax.inject.Singleton
 import org.hl7.fhir.r4.model.Resource
+import org.hl7.fhir.r4.model.ResourceType
 
 @Singleton
 class QuestXFhirQueryResolver @Inject constructor(val fhirEngine: FhirEngine) : XFhirQueryResolver {
   override suspend fun resolve(xFhirQuery: String): List<Resource> {
-    return fhirEngine.search(xFhirQuery)
+    Log.d("FIKRI XFHIRQUERY", xFhirQuery)
```

Comment:
Do we need the commented lines of code in this function?

Reply:
We don't need it